pyproject.toml

```diff
@@ -43,7 +43,7 @@ requires = [
 
 [project]
 name = "scipy"
-version = "1.15.0rc1"
+version = "1.15.0rc2"
```

Comment:
I think this is now just a one line change given new version handling changes, compared to previous approach at gh-20839.

That said, this may not be quite right, since `rc2` proper happens after the additional backports. I've typically just done this, but more recently I think we've been more formally correct by appending `.dev0` in these cases perhaps?

Reply:
Yeah adding `.dev0` is a little more correct, but in practice it doesn't matter much. I'm happy either way.